openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -224,7 +224,7 @@ Figure: Issuance using Authorization Code Flow
 
 (1b) The Issuer-initiated flow begins as the Credential Issuer generates a Credential Offer for certain Credential(s) that it communicates to the Wallet, for example, as a QR code or as a URI. The Credential Offer contains the Credential Issuer's URL and the information about the Credential(s) being offered. This step is defined in (#credential_offer).
 
-(2) The Wallet uses the Credential Issuer's URL to fetch the Credential Issuer metadata as described in (#credential-issuer-metadata). The Wallet needs the metadata to learn the Credential types and formats that the Credential Issuer supports, and to determine the Authorization Endpoint (OAuth 2.0 Authorization Server) as well as Credential Endpoint required to start the request. This specification enables deployments where the Credential Endpoint and the Authorization Endpoint are provided by different entities. Please note that in this example the Credential Issuer and OAuth 2.0 Authorization Server correspond to the same entity.
+(2) The Wallet uses the Credential Issuer's metadata as described in (#credential-issuer-metadata) to learn what Credential types and formats the Credential Issuer supports and to determine the issuer URL of the OAuth 2.0 Authorization Server the Credential Issuer relies on. In this example the Credential Issuer provides the OAuth 2.0 Authorization Server. However, this specification allows for deployments where the Credential Issuer API and the Authorization Server are separate services, potentially provided by different entities.
```

Comment:
```suggestion
(2) The Wallet uses the Credential Issuer's metadata as described in (#credential-issuer-metadata) to learn the Credential Endpoint, what Credential types and formats the Credential Issuer supports and to determine the issuer URL of the OAuth 2.0 Authorization Server the Credential Issuer relies on. In this example the Credential Issuer provides the OAuth 2.0 Authorization Server. However, this specification allows for deployments where the Credential Issuer API and the Authorization Server are separate services, potentially provided by different entities.
```

Reply:
@paulbastian ^

Reply:
```suggestion
(2) The Wallet uses the Credential Issuer identifier to fetch the Credential Issuer metadata, as described in (#credential-issuer-metadata). The Wallet needs this metadata to learn about the capabilities of the Credential Issuer, including their supported Credential formats and types. These capabilities also encompass the endpoints for both the Credential Issuer and the Authorization Server (OAuth2) to initiate the request. This specification facilitates deployments where the endpoints for the Credential Issuer and the Authorization Server are provided by different entities. Please note that, in this example, the Credential Issuer and OAuth 2.0 Authorization Server correspond to the same entity.
```

Reply:
I like the suggestions from @Sakurann and @awoie, but I would like to adapt it a bit, e.g. `Credential Issuer identifier` is capitalized, i.e. `Credential Issuer Identifier`. Imo, it is better to remove the part about starting/initiating the request with the same argument that @peppelinux mentioned [above](https://github.com/openid/OpenID4VCI/pull/148/files#r1428047507): You do not need the endpoints of the Credential Issuer in this flow to "start/initiate the request". The first request in the flow is the Authorization Request (aside from fetching the metadata...).

```suggestion
(2) The Wallet uses the Credential Issuer Identifier to fetch the Credential Issuer metadata as described in (#credential-issuer-metadata). The Wallet needs this metadata to learn about the capabilities of the Credential Issuer, including the supported Credential formats and types. These capabilities also encompass the endpoints of both the Credential Issuer and the OAuth 2.0 Authorization Server. This specification facilitates deployments where the endpoints of the Credential Issuer and the Authorization Server are provided by different entities. Note that in this example the Credential Issuer and OAuth 2.0 Authorization Server correspond to the same entity.
```